config/modules/openjtalk.conf

```diff
@@ -0,0 +1,20 @@
+# dictionary directory (open_jtalk -x option)
+OpenjtalkDictionaryDirectory "/var/lib/mecab/dic/open-jtalk/naist-jdic"
+
+# HTS voice file (open_jtalk -m option)
+OpenjtalkVoice "/usr/share/hts-voice/nitech-jp-atr503-m001/nitech_jp_atr503_m001.htsvoice"
```

Comment:
Will there be other voices? It would be useful to add the series of known possibilities

Reply:
BTW, does the jtalk tool provide a way to get information from a voice file? It'd be better to be able to just configure a directory and let the module look around inside and find the available voices to properly list them.

Reply:
I apologize for the late reply.
To the best of my knowledge, open_jtalk doesn't provide a feature to get information from voice files.
Therefore, the most feasible approach would likely be to display the filenames of the .htsvoice files in the specified directory as voice names.

Reply:
Ok, then a way is to list the available possibilities in the config file with a Dependency line, similarly to the `VoiceFileDependency` option used by the mbrola generic modules. That way the module can check which voices are actually available, while still allowing the user to add more voices to the list.

Reply:
OK, I will check the implementation of the mbrola module.

Reply:
the mbrola generic module, more precisely, i.e. the generic module with the `espeak-mbrola-generic.conf` configuration file